cahiers-d-ethnomusicologie.csl

```diff
@@ -0,0 +1,484 @@
+          </else-if>
+          <else>
+            <text macro="translator"/>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <text term="anonymous" text-case="uppercase"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author-citation">
+    <choose>
+      <if variable="author editor translator" match="any">
+        <names variable="author">
+          <name form="short"/>
+          <substitute>
+            <names variable="editor"/>
+            <names variable="translator"/>
+          </substitute>
+        </names>
+      </if>
+      <else>
+        <text term="anonymous" text-case="uppercase"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="container-author">
+    <names variable="container-author">
+      <name and="text" name-as-sort-order="all" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
+      </name>
+    </names>
+  </macro>
+  <macro name="container-responsability">
+    <choose>
+      <if variable="container-author editor translator" match="any">
+        <choose>
+          <if variable="container-author">
+            <text macro="container-author"/>
+          </if>
+          <else-if variable="editor">
+            <text macro="editor"/>
+          </else-if>
+          <else>
+            <text macro="translator"/>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <text term="anonymous" text-case="uppercase"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year" form="long" suffix=" "/>
+        </date>
+      </if>
+      <else>
+        <text term="no date"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="book thesis map motion_picture song manuscript" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else-if type="paper-conference speech chapter article-journal article-magazine article-newspaper post-weblog post webpage broadcast" match="any">
+        <text variable="title" prefix="« " suffix=" ». "/>
+        <choose>
+          <if variable="container-author editor translator" match="any">
+            <text macro="container-responsability"/>
+            <choose>
+              <if variable="container-title event" match="any">
+                <text value=", "/>
+              </if>
+            </choose>
+          </if>
+        </choose>
+        <choose>
+          <if variable="container-title">
+            <text variable="container-title" font-style="italic"/>
+          </if>
+          <else>
+            <text variable="event" font-style="italic"/>
+          </else>
+        </choose>
+      </else-if>
+      <else-if type="report">
+        <text variable="number" suffix=" : "/>
```

Comment:
Yes. You’re probably right. There’s a problem here, which might come back later on.
The journal’s explicit guidelines are quite minimal. I sent the link in a previous comment. When the editors have to deal with documents which are not stated in the guidelines they try to do "something sensible", usually something which will look like the other references. 
Now I started this style from a template which claims to follow the ISO 690 norm for French bibliographies (its link is in the `<info>`). The template covers far more document types than those addressed in the journal guidelines. I hesitated between keeping and removing those extra types, and finally opted for keeping them, in case they come in handy some day. But quite frankly, I sometimes don’t know how people use those types. "Report" is an example of this: I don’t use it and never encountered it in my colleagues’ or my students’ bibliographies. I have no idea what the report "number" would represent in our discipline. That being said, I agree that in case someone uses it, putting a number right after the title doesn’t really make sense in this style. Following your suggestion, I then migrated the number to the description, between the "edition" and the "publisherplace".
If you prefer to deal with this kind of things — I mean document types which are not explicitly addressed in the journal’s guidelines — in another way, please let me know.